### PR TITLE
Add UpdateIndexSettingsRequest

### PR DESCRIPTION
**Pull request by SHyx0rmZ**

### CODE REVIEW

**Comment by da-wen** [comment]
Can you write unit and sandbox tests for this ? Plz :-)

File: src/Request/V090x/Index/UpdateIndexSettingsRequest.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    /**
+     * Gets a response class name that is supported by this class
+     *
+     * @return string
+     * @author Daniel Wendlandt
+     * @author Patrick Pokatilo <mail@shyxormz.net>
+     */
+    public function getSupportedClass()
+    {
+        return IndexResponse::class;
```
Comment (xenji) [inline src/Request/V090x/Index/UpdateIndexSettingsRequest.php]: @SHyx0rmZ This is a PHP 5.5 only feature. Please use a PHP 5.4 compatible version of returning a class name.

**Comment by xenji** [comment]
@SHyx0rmZ Please check your PR for any features requiring PHP 5.5 or later. I've marked one of the usages of `::class`, but this happens in every request file. Please correct this to make the lib still usable with PHP 5.4.

**Comment by SHyx0rmZ** [comment]
I'm sorry, I already noticed but haven't come around to fixing it, yet. Will do so shortly.

**Pull request merged**